src/database-controller/src/poller/index.js

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+require('module-alias/register');
+require('dotenv').config();
+const AsyncLock = require('async-lock');
+const { Sequelize } = require('sequelize');
+const Op = Sequelize.Op;
+const DatabaseModel = require('openpaidbsdk');
+const logger = require('@dbc/core/logger');
+const { Snapshot, AddOns, synchronizeRequest } = require('@dbc/core/framework');
+const interval = require('interval-promise');
+const config = require('@dbc/poller/config');
+const fetch = require('node-fetch');
+const { deleteFramework } = require('@dbc/core/k8s');
+// maxPending is set to 1 to avoid the queue to be too long.
+// If any framework is not synced/deleted, it will be synced/deleted in the next polling round.
```

Comment:
> polling round. [](start = 80, length = 15)

in the next polling round, so not need to explicitly retry on error

Reply:
commented.